src/components/ListItem.tsx

```diff
@@ -83,6 +84,30 @@ export function ListItemCheckBox({ item, listPath }: Props) {
 		}
 	};
 
```

Comment:
suggestion: i think we could move this logic for updating the item quantity into the `ItemQuantityForm`. reading through the code right now it feels liek tehre is some duplication of logic, kind of adding logic to components to make another component work. 

moving this to be handled by the `ItemQuantityForm` it could make it a bit more reusable because we could just place it into the location it is needed and not write logic within that component to get it to work. it owuld be like this check box and other forms the logic to do the updating and steps the form is doing is housed within the logic.

To get the `item` prop needed with these changes on the list view we could try moving the `<ItemQuantityForm saveItemQuantity={editItemQuantity} item={item} /> x{" "}` call into the `List` component with this logic ```{filteredListItems.map((item) => (<ListItemCheckBox key={item.id} item={item} listPath={listPath} />))}``` and I believe it would get the same result.

Reply:
Great idea, that way the logic outside of firebase for quantity is all referable in one place.